dockerfile/visualizer/Dockerfile

```
@@ -14,11 +14,16 @@ RUN groupadd --gid $USER_GID $USERNAME && \
     echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME && \
     chmod 0440 /etc/sudoers.d/$USERNAME
 
+# Update nodejs version
+RUN apt-get update \
+    && apt install curl \
+    && curl -sL https://deb.nodesource.com/setup_12.x -o nodesource_setup.sh \
```

```suggestion
    && curl -sL https://deb.nodesource.com/setup_12.x -o nodesource_setup.sh \
    && bash nodesource_setup.sh
```

Need to execute the script

---

Thanks for checking. I forgot to commit that line.